WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -46,13 +78,114 @@ private extension ReviewsDashboardCard {
             .disabled(viewModel.syncingData)
         }
     }
+
+    var reviewsFilterBar: some View {
+        HStack {
+            AdaptiveStack(horizontalAlignment: .leading) {
+                Text(Localization.status)
+                    .foregroundStyle(Color(.text))
+                    .subheadlineStyle()
+
+                Text("All") // TODO: dynamically change based on filter selection
+                    .subheadlineStyle()
+            }
+            Spacer()
+
+            Menu {
+                ForEach(viewModel.filters, id: \.self) { filter in
+                    Button {
+                        // TODO
+                    } label: {
+                        SelectableItemRow(title: filter.title, selected: false)
+                    }
+                }
+            } label: {
+                Image(systemName: "line.3.horizontal.decrease")
+                    .foregroundStyle(Color.secondary)
+            }
+        }
+    }
+
+    func ReviewRow(for review: ProductReview, isLastItem: Bool) -> some View {
+        HStack(alignment: .top, spacing: 0) {
+            Image(systemName: "bubble.fill")
+                .foregroundStyle(review.status == .hold ? Color.secondary : Color(.wooCommercePurple(.shade60)))
```

Comment:
I just discovered that the bubble color change depends on the review's read state, instead of pending state. I'll update this in the next functionality PR.